app/views/static_pages/_donate_content.html.erb

```diff
@@ -1,21 +1,18 @@
-<div class="col-md-8 ps-md-4 bigger pb-5 pb-md-0">
```

Comment:
So this partial is being used on `/donate` and on `/adopter_fosterer/donations` so we need to keep this partial the way it was so that it renders with proper styling on each route.